### ISSUE TITLE
Warning: ListView is deprecated and will be removed in a future release. 

### ISSUE THREAD

**Issue by diegolaciar** [cooperka/react-native-immutable-list-view#41]
Hi I get this warning and don' t know way?

"Warning: ListView is deprecated and will be removed in a future release. "

I'm using expo sdk 35.0.0

I just replaced my Flatlist  with this code
```
 <ImmutableVirtualizedList
      immutableData={messageList}
      renderItem={renderItemImmutable}
    />
```

**Comment by cooperka** [comment]
Hi @diegolaciar, are you sure you aren't using `ImmutableListView` or regular `ListView` anywhere else in your app? ~I'm not sure why you'd be seeing this warning otherwise.~

If you link to an example app with this issue I can look into it further. Thanks.

**Comment by diegolaciar** [comment]
Hi @cooperka ,

Yes I'm sure I'm not using ListView anywhere else.

Also I wanted to comment I'm not able to import VirtualizedList from react-native.  I don't know why or if it' s because I' m using using expo sdk 35.0.0 and typescript.

**Comment by cooperka** [comment]
My mistake, `ImmutableListView` was being exported from the main `index.js` file and therefore being included automatically even if you were only using `ImmutableVirtualizedList`.

This has been fixed in the latest v2.0 release; please upgrade to get rid of the warning. Thanks for the report!
